src/MarginTrading.Frontend/Repositories/Contract/IMaintenanceInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace MarginTrading.Frontend.Repositories.Contract
+{
+    public interface IMaintenanceInfo
```

Comment:
Better to split interfaces and implementation to separate files.
And it's not a Repository actually, it's more like an entity, so better to move it to Entities folder
Found in multiple places.